suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -394,25 +394,28 @@ export const startDiscoveryThunk = createThunk(
         }
 
         const { deviceState, authConfirm } = discovery;
-        const metadataEnabled = metadata.enabled && !device.metadata[1]; // todo: can't import constant
+
+        // todo: I can't import from packages/suite here I guess
+        // eslint-disable-next-line no-restricted-syntax
+        const encryptionVersion = getState().suite.settings.experimental?.includes(
```

Comment:
I think proper solution here is to pass selector through `extra` and use it here.

Reply:
Lately we have added `selectSuiteSettings` selector to extras, I think this is very similar use-case.